src/manifolds/Hyperbolic.jl

```diff
@@ -21,91 +21,193 @@ Note that while the [`MinkowskiMetric`](@ref) renders the [`Lorentz`](@ref) mani
 pseudo-Riemannian, on the tangent bundle of the Hyperbolic space it induces a Riemannian
 metric. The corresponding sectional curvature is $-1$.
 
+If `p` and `X` are `Vector`s of length `n+1` they are assumed to be
+a [`HyperboloidPoint`](@ref) and a [`HyperboloidTVector`](@ref), respectively
+
+Other models are the Poincaré ball model, see [`PoincareBallPoint`](@ref) and [`PoincareBallTVector`](@ref), respectiely
+and the Poincaré half space model, see [`PoincareHalfSpacePoint`](@ref) and [`PoincareHalfSpaceTVector`](@ref), respectively.
+
 # Constructor
 
     Hyperbolic(n)
 
-Generate the $ℍ^{n} ⊂ ℝ^{n+1}$
+Generate the Hyperbolic manifold of dimension `n`.
 """
 struct Hyperbolic{N} <: AbstractEmbeddedManifold{ℝ,DefaultIsometricEmbeddingType} end
 
 Hyperbolic(n::Int) = Hyperbolic{n}()
 
+@doc raw"""
+    HyperboloidPoint <: MPoint
+
+In the Hyperboloid model of the [`Hyperbolic`](@ref) $ℍ^n$ points are represented
```

Comment:
Using ℍ^n for the hyperbolic manifold clashes with our symbol for quaternions.

Reply:
It would be a good idea to document our usage of ℍ in the notation section of documentation.

Reply:
Oh, I just copied that from the initial docstring of `Hyperbolic`, but you're right; that's a conflict. That's sad, since I like the symmetry of the Sphere and Hyperbolic both having the math Capital letters.

Reply:
I changed the hyperbolic to `$\mathcal H$` and added real, complex, quaternions to the notations list.